tests/integration/test_cloudformation.py

```diff
@@ -2174,6 +2174,9 @@ def test_cfn_with_apigateway_resources(self):
 
         self.assertEqual(len(resources), 1)
 
+        # assert request parameter is present in resource method
+        self.assertEqual(resources[0]['resourceMethods']['POST']['requestParameters'],
+            {'method.request.path.account': True})
```

Comment:
Thanks for this PR @KrishnanRanjithkumar ! 👍 In addition to this CoudFormation test, would be great if we could extend the tests in `test_apigateway.py` to include a simple end-to-end test that actually passes a request parameter via API Gateway to a target endpoint (e.g., using `HTTP_PROXY` integration). Thanks!